### ISSUE TITLE
do not use this repo ?

### ISSUE THREAD

**Issue by mariya12290** [Yao-Shao/Waymo_Kitti_Adapter#11]
Hey all,

please do not use this repo. conversion is entirely wrong. not just in ground truth, but also in lots of cases.

**Comment by MartinHahner** [comment]
@mariya12290: Is this one better / correct?
https://github.com/caizhongang/waymo_kitti_converter

Or did you find another one?

**Comment by mariya12290** [comment]
@MartinHahner waymo_kitti_converter is good. currently I am using it. But still one has to go through all the data set after the conversion by writing python script especially to find missing data/frame, especially in label folder. you can get to know if there are any missing data in each folder just using ls | wc in linux terminal.

**Comment by SofianeB-03** [comment]
Hello @mariya12290, did you make predictions with the conversion tool: https://github.com/caizhongang/waymo_kitti_converter ?
Because I have tried to make detections (with https://github.com/maudzung/SFA3D) on the waymo converted data to kitti but I have wrong results.
Thank you

**Comment by mariya12290** [comment]
Hey @SofianeB-03 I trained on waymo data set from scratch, I am getting the predictions, but lots of false positive. I do not know the exact reason, but waymo data is sequence data, so it makes the model to over fit on train data(this is my understanding), even though we feed the data randomly.  

If you know any other reasons, please let me know

Thank you

**Comment by SofianeB-03** [comment]
@mariya12290, thank you very much for your reply. I agree with you, I think that sequences data can make overfitting. But have you trained your model on another dataset (for example KITTI) and then tested it on Waymo data ?

**Comment by mariya12290** [comment]
@SofianeB-03 I tried, but no good predictions, all are false positive. that is the problem with Lidar. The reason may be position of Lidar and also as Andrew Nj mentioned in his lecture videos, distribution of data(which means, kitti data is from Germany and Waymo data is from USA), but I am not sure about this. It is just my assumption.

**Comment by SofianeB-03** [comment]
this is really useful information that I didn't know. Thanks a lot!

**Comment by SofianeB-03** [comment]
@mariya12290 Thank you again for taking the time to respond me.
